app/views/articles/_home_page.html.erb

```diff
@@ -56,8 +56,7 @@
       <div class="panel panel-default">
         <div class="panel-body">
           <h2>Search tip</h2>
-          <h4><%= t("searchworks.articles.search_tips.heading") %></h4>
-          <p><%= t("searchworks.articles.search_tips.body") %></p>
+          <%= render SearchTips.random %>
```

Comment:
So this is pretty static and straight forward right now.

Another thought was to render all the tips into the page and use javascript to randomize.  That would have the benefit of allowing some sort of rotation/carousel of tips or a control to update with a new random tip (although that's not necessarily in the design so may be overkill at this time).